src/components/results-list/Search-results-list.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Animal, AnimalsPagedQueryResponse } from '../../services/api';
+import './Search-results-list.css';
+
+interface SerachResultsListProps {
+  data?: AnimalsPagedQueryResponse;
+}
+
+export class SearchResultsList extends React.Component<SerachResultsListProps> {
+  constructor(props: SerachResultsListProps) {
+    super(props);
+  }
+
+  getAnimalType(animal: Animal) {
+    const type: string[] = [];
+    if (animal.avian) {
+      type.push('avian');
+    }
+    if (animal.canine) {
+      type.push('canine');
+    }
+    if (animal.earthAnimal) {
+      type.push('earth animal');
+    }
+    if (animal.earthInsect) {
+      type.push('earth insect');
+    }
+    if (animal.feline) {
+      type.push('feline');
+    }
+    return type.join(', ');
+  }
+
+  private renderListHeader() {
+    return (
+      <li key="0" className="search-results__list-item header">
```

Comment:
you don't need this key. keys is necessary only in array mapping, and some very specific cases

Reply:
Fixed.